bond/bond.go

```diff
@@ -68,7 +68,7 @@ func loadConfigFile(bytes []byte) (*bondingConfig, string, error) {
 }
 
 // retrieve the link names from the bondConf & check they exist. return an array of linkObjectsToBond & error
-func getLinkObjectsFromConfig(bondConf *bondingConfig, netNsHandle *netlink.Handle) ([]netlink.Link, error) {
+func getLinkObjectsFromConfig(bondConf *bondingConfig, netNsHandle *netlink.Handle, isDel bool) ([]netlink.Link, error) {
```

Comment:
Is there any chance to split this into two different functions to eliminate the boolean parameter? Boolean parameters look mysterious from the call site.
Maybe something like `getLinkObjectsFromConfig(...)` and `getLinkObjectsFromConfigIgnoringMissing(...)`. WDYT?

Reply:
maybe, however, I preferred this approach which was reviewed and approved upstream.